PypeS/pypepad.py

```diff
@@ -276,7 +276,7 @@ def GetSuggestionsList(self,word):
             scriptindex = self.text_input.search('vmtk',self.wordIndex[0],backwards=1)
             moduleName  = self.text_input.get( scriptindex,scriptindex+' wordend' )
             try:
-                exec('import '+moduleName)
+                exec('from . import '+moduleName)
                 exec('scriptObjectClassName =  '+moduleName+'.'+moduleName)
```

Comment:
Ahah :-) This library was one of the first things I wrote in Python a loong time ago. At the time I had just learned object oriented programming from a C++ book, maybe that's what you're picking up. Or maybe it's just weird in its own self 😄 

My suggestion here is to change:
```
    scriptObjectClasses = [x for x in dir(module) if isclass(getattr(module, x))]
```
to
```
    scriptObjectClasses = [x for x in dir(module) if isclass(getattr(module, x)) and issubclass(getattr(module, x), pypes.pypeScript)]
```
This would guarantee we don't bump into some extra classes defined in the module when we get `scriptObjectClassName = scriptObjectClasses[0]`.